src/main/kotlin/racingcar/domain/Position.kt

```diff
@@ -1,5 +1,5 @@
 package racingcar.domain
 
-class Position(val position: String) {
-    fun accelerate() = Position("$position-")
+class Position(val position: Int = 0) {
+    fun accelerate() = Position(position + 1)
 }
```

Comment:
Position 인스턴스가 값을 표현하는 그 이상, 이하가 아니라고 생각이 드는데 Data Class도 한번 생각해볼 만한거 같아!
그렇다면 코드가

```suggestion
data class Position(val position: Int = 0) {
    fun accelerate() = this.copy(position + 1) // 이 코드는 에러가 안나는지는 확실히 모르겠다. 지금 그냥 Github에서 바로 쓰고 있어서 ㅋㅋㅋ
}
```

Reply:
아.. 데이터 클래스 저번에 너한테 이야기 들은거 기억난다 ㅋㅋㅋ
현재로썬 확장 될 여지가 없기 때문에 data class로 두는게 좋을거같네